packages/vm/src/instance.rs

```diff
@@ -62,15 +62,22 @@ where
         options: InstanceOptions,
         memory_limit: Option<Size>,
     ) -> VmResult<Self> {
-        let module = compile(code, memory_limit)?;
-        Instance::from_module(&module, backend, options.gas_limit, options.print_debug)
+        let module = compile(code, memory_limit, &[])?;
+        Instance::from_module(
+            &module,
+            backend,
+            options.gas_limit,
+            options.print_debug,
+            None,
+        )
     }
 
     pub(crate) fn from_module(
         module: &Module,
         backend: Backend<A, S, Q>,
         gas_limit: u64,
         print_debug: bool,
+        extra_imports: Option<HashMap<&str, Exports>>,
```

Comment:
Just realized that can be a little simpler, without the explicit internal `HashMap`.